### PR TITLE
Fix rename of columns after a summarise

### PR DESCRIPTION
**Pull request by philss**
We can´t be sure the order of columns after a summarise is performed by
polars. So using a proper `rename` makes sure that we are rename the
correct - named - column.

Fixes https://github.com/elixir-nx/explorer/issues/310

### CODE REVIEW

File: test/explorer/data_frame/grouped_test.exs [hunk 1]
```diff
@@ -109,6 +109,26 @@ defmodule Explorer.DataFrame.GroupedTest do
              }
     end
 
+    test "pivote_wider and then summarise with a rename" do
```
Comment (josevalim) [inline test/explorer/data_frame/grouped_test.exs:112]:
```suggestion
    test "pivot_wider and then summarise with a rename" do
```

**Comment by josevalim** [comment]
Maybe as a fix it would be better if we guaranteed that summarize has the given order? Or is that not possible? We may have a similar issue with mutate too, as it has a similar API.

**Comment by josevalim** [comment]
Apparently polars does not guarantee an order, so I think this is the correct approach. I will just push a tiny improvement.

**Review by josevalim — APPROVED** [review]
The previous code was optimized to rename all columns. Now that we always pass `[{old, new}]`, I optimized it to this new version. :)

**Comment by philss** [comment]
@josevalim awesome! I can't approve my own PR, but I assume I can merge now, right? :smile:

**Pull request merged**